src/oneclick/analysis/trackAnalysis.py

```diff
@@ -48,7 +48,7 @@ def run(cls, config:Config):
                                 #if len(find_in_list(line,p.log))==0:
                                 p.log.append(line)
                             if process.returncode == 0:
-                                p.status = 'OK'
+                                p.status = 'Complete'
```

Comment:
I am ok with updating the status to Complete, keep in mind that this is also being used to decide if the analysis needs to be run on restart.  Please make sure this part of the code has also been addressed.